lib/SIL/Verifier/SILOwnershipVerifier.cpp

```diff
@@ -774,7 +882,8 @@ void SILInstruction::verifyOperandOwnership() const {
 static void
 verifySILValueHelper(const SILFunction *f, SILValue value,
                      LinearLifetimeChecker::ErrorBuilder &errorBuilder,
-                     DeadEndBlocks *deadEndBlocks) {
+                     DeadEndBlocks *deadEndBlocks,
+                     ReborrowVerifier &reborrowVerifier) {
```

Comment:
Does the reborrow verifier need to be known about outside of the SILValueOwnershipChecker?

Reply:
Yes. This is for the common visitedOps and visitedPhiArgs maps in ReborrowVerifier. That way we won't check multiple times and generate duplicate errors.

Reply:
I am not convinced. I need to understand this more.

Reply:
Maybe these examples will help : 
We don't want to re-verify an already visited phiArg with same baseValue when processing through another path. Ex : `borrowed_owned_val_not_dominate_correct` in `borrow_validate.sil`

We don't want to raise duplicate errors when this happens. Ex : `borrowed_owned_val_not_dominate_incorrect` in `borrow_validate.sil`

Reply:
I am fine with this for now. The important thing to me was that this verification did not leak into the normal SILVerifier.